File: src/codes/duplicate-logic/ts/ksrc.ts
class ImageUploader {
  upload(file: string) {
    console.log("検証中...");
    console.log(`画像ファイルをアップロード中: ${file}`);
    console.log("完了しました");
  }
}

class VideoUploader {
  upload(file: string) {
    console.log("検証中...");
    console.log(`動画ファイルをアップロード中: ${file}`);
    console.log("完了しました");
  }
}

class AudioUploader {
  upload(file: string) {
    console.log("検証中...");
    console.log(`音声ファイルをアップロード中: ${file}`);
    console.log("完了しました");
  }
}

File: src/codes/duplicate-logic/py/ksrc.py
class ImageUploader:
    def upload(self, file: str):
        print("検証中...")
        print(f"画像ファイルをアップロード中: {file}")
        print("完了しました")

class VideoUploader:
    def upload(self, file: str):
        print("検証中...")
        print(f"動画ファイルをアップロード中: {file}")
        print("完了しました")

class AudioUploader:
    def upload(self, file: str):
        print("検証中...")
        print(f"音声ファイルをアップロード中: {file}")
        print("完了しました")
